Libraries/Components/Button.js

```diff
@@ -25,107 +25,114 @@ import type {ColorValue} from '../StyleSheet/StyleSheet';
 
 type ButtonProps = $ReadOnly<{|
   /**
-   * Text to display inside the button
+    Text to display inside the button. On Android the given title will be converted to the uppercased form.
    */
   title: string,
 
   /**
-   * Handler to be called when the user taps the button
+    Handler to be called when the user taps the button. The first function argument is an event in form of [PressEvent](pressevent).
    */
   onPress: (event?: PressEvent) => mixed,
 
   /**
-   * If true, doesn't play system sound on touch (Android Only)
-   **/
+    If `true`, doesn't play system sound on touch.
+
+    @platform android
+
+    @default false
+   */
   touchSoundDisabled?: ?boolean,
 
   /**
-   * Color of the text (iOS), or background color of the button (Android)
+    Color of the text (iOS), or background color of the button (Android).
+
+    @default android: #2196F3, ios: #007AFF
```

Comment:
We can extract both the default values from key value pair (sort of) format. Does this look good?

Reply:
1. Note that the values in the Markdown are `'#2196F3'` and `'#007AFF'` (in quotes)
2. The format seems reasonable to me, but maybe we can go with something like this (syntax inspired by JSDoc "inline tags") where `@default` can be qualified by a `@platform` tag:
```
@default {@platform android} '#2196F3'
@default {@platform ios} '#007AFF'
```

Reply:
These are the react-docgen outputs on the file when using the two formats: 

### String format: 
`      "description": "Color of the text (iOS), or background color of the button (Android).\n\n    @default android: #2196F3, ios: #007AFF"
`

### JSDocs format:
`      "description": "Color of the text (iOS), or background color of the button (Android).\n\n    @default {@platform android} '#2196F3'\n    @default {@platform ios} '#007AFF'"`

Are we going to parse the values from these strings generated using react-docgen? If so, which one would be better approach to use for parsing?

Reply:
From code readability point of view the JSDoc format is far more readable. I'm not sure which is easier to parse in `react-docgen`, but since the parsing code needs to written only once, and the comments need to be read, written and maintained continuously, readability is more important than parseability.

I'd say let's move forward with:
```
@default {@platform android} '#2196F3'
@default {@platform ios} '#007AFF'
```

Thorough work on this, everyone 👍

Reply:
@jevakallio Okay thanks. That's sorted then.